core/transaction/pool/pool.go

```diff
@@ -43,14 +38,18 @@ func (tp *TxnPool) AppendTxnPool(txn *Transaction) ErrCode {
 		log.Info("Transaction verification failed", txn.Hash())
 		return errCode
 	}
+	var txnList []*Transaction
+	for _, tx := range tp.txnList {
+		txnList = append(txnList, tx)
+	}
+	if errCode := VerifyTransactionWithBlock(txnList); errCode != ErrNoError {
```

Comment:
this will cost a lot of time to verify a transaction, when the txpool is very large.

Reply:
do you have ideas how this can be fixed? I thought to make `VerifyTransactionWithBlock` accept map instead of array but it's called also in `core/ledger/validator.go` and there is array

Reply:
ok, I think I fixed this @NoelBright